core/src/main/java/fj/P2.java

```diff
@@ -1,8 +1,10 @@
 package fj;
 
 import static fj.Function.*;
-
+import static fj.data.optic.PLens.pLens;
```

Comment:
@mperry so... do you suggest to put optics instances in Prism, Lens, Ptraversal and the like...? I think this could be practical...
if they are isolated in their own package, those 'orphan' optics that will be harder to find which will lower their practicality.